### PR TITLE
Login/Register styling

### PR DESCRIPTION
**Pull request by AndrewA0112**
Minor changes to MainPage to try to fix styling, seems header doesn't act correctly with the styling

### CODE REVIEW

File: didact/src/components/login/Login.jsx [hunk 1]
```diff
@@ -4,75 +4,79 @@ import * as Yup from 'yup';
...
+    //     "& *":{
+    //         height:'100%',
+    //         backgroundColor: '#5A5A5A',
+    //         color: 'white',
+    //         border: 'transparent',
+    //         borderRadius: '5%',
+    //         width: '33%',
+    //         textDecoration: 'none'
+    //     }
+    // }
```
Comment (Bastlifa) [inline didact/src/components/login/Login.jsx]: All these comments, learn to trust the commit process :)
j/k, I don't know how to do all that stuff either. We should have like, 2 weeks on just git.

File: didact/src/components/login/Login.jsx [hunk 2]
```diff
@@ -84,26 +88,34 @@ const LoginForm = (props) => {
...
+                        <button type="submit">Login</button>
                     </div>
                 </div>
-                <div className={classes.buttonContainer}>
-                <button className={classes.loginButton} type="submit">Login</button>
-                </div>
             </Form>
-        </>
+            <div className="socialButtons">
+                <a href="http://didactlms-staging.herokuapp.com/api/auth/facebook">Sign In With Facebook</a>
```
Comment (Bastlifa) [inline didact/src/components/login/Login.jsx]: I'd make this URL dynamic env. Set on netlify/local machine.

File: didact/src/components/mainPage/PageStyles.js [hunk 3]
```diff
@@ -3,4 +3,13 @@ import styled from 'styled-components'
...
     width: auto;
+
+    .drawer {
+        /* padding-right: 50px; */
+    }
+
+    .headerMain {
+        /* padding-left: 50px; */
+        width : 100%
+    }
```
Comment (Bastlifa) [inline didact/src/components/mainPage/PageStyles.js]: no newline at end of file

File: didact/src/components/register/Register.jsx [hunk 4]
```diff
@@ -1,9 +1,13 @@
-import React from "react";
-import {Form, Field, withFormik} from "formik";
+import React, { useState } from "react";
```
[the remark this answers is not in the document]
Comment (AndrewA0112) [reply didact/src/components/register/Register.jsx]: Oopsie, must of forgot when i was testing confirmPassword

**Pull request merged**